src/tester.tsx

```diff
@@ -44,22 +47,26 @@ const NullComponent = (props) => <Fragment {...props} />;
  * @returns {Tester}
  */
 class Tester {
-  config;
-  initialMount;
-  onBeforeMount;
-  opts;
-  profile;
-  props;
-  shallow;
-  TestedComponent;
-  wrapper;
-  wrappers;
-
-  constructor (TestedComponent, opts = {}) {
-    this.config = this.constructor.Configuration;
-    this.opts = opts;
+  static Configuration: ConfigurationClass;
+
+  public opts: ITesterOps;
+
+  public config: ConfigurationClass;
+  public initialMount: React.ReactNode;
+  public onBeforeMount?: (tester: Tester) => Promise<void>;
+  public profile: IProfile;
+  public props: object;
+  public TestedComponent: ComponentClass;
+
+  public AppState: any;
+  public shallow: any;
+  public wrapper: any;
+
+  constructor (TestedComponent: ComponentClass, opts: ITesterOps = {}) {
```

Comment:
`ITesterOps` I now see that `Ops` was intentional or an amazingly well achieved copy/pasting :)